### PR TITLE
2019 Lexus NX FW versions

### PR DESCRIPTION
**Pull request by flightguy2**
Adding fingerprint only for 2019 Lexus NX.  Currently, only 2018 & 2020 NX exists.

### CODE REVIEW

File: selfdrive/car/toyota/values.py [hunk 1]
```diff
@@ -55,6 +55,7 @@ class CAR:
   LEXUS_ESH_TSS2 = "LEXUS ES HYBRID 2019"
   LEXUS_IS = "LEXUS IS 2018"
   LEXUS_NX = "LEXUS NX 2018"
+  LEXUS_NX = "LEXUS NX 2019"
```
Comment (vanillagorillaa) [inline selfdrive/car/toyota/values.py:58]: This isn't needed and should be removed. The year doesn't matter as long as the platform is the same

**Comment by vanillagorillaa** [comment]
Please provide the dongleID for the device this was taken from

File: selfdrive/car/toyota/values.py (hunk 1 above)
Comment (adeebshihadeh) [reply selfdrive/car/toyota/values.py:58]:
```suggestion
```

**Comment by flightguy2** [comment]
Dongle ID:  f61143b2bcdedf19

Regards,

Steve

**Comment by flightguy2** [comment]
> 

Original route used to fingerprint was from several weeks ago, but I believe one of the more recent route segments should demonstrate the same fingerprint values.  I hope this works...

Segment:  f61143b2bcdedf19|2021-12-02--13-41-54

**Pull request merged**